### PR TITLE
fix: missing config in wds widgets

### LINKED ISSUE #31223: Analyse WDS widgets configs
It is necessary to analyse the widgets configs and add/remove the configuration of the properties. 

Related thread: https://theappsmith.slack.com/archives/C04P60V0VQE/p1708448327343889

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #31223 

## Automation
/ok-to-test tags="@tag.Widget"

## Summary by CodeRabbit

- **New Features**
	- Introduced configuration settings for Button, IconButton, Input, Checkbox Group, Currency Input, Phone Input, Radio Group, Stat Box, Table, and Toolbar Buttons widgets, enhancing customization options such as size, visibility, text, disabled state, and more.
- **Refactor**
	- Reorganized and consolidated import paths and references across various widgets for improved code maintainability and readability.
- **Chores**
	- Updated and restructured configuration files and imports for a more streamlined development experience.

> [!IMPORTANT]  
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/8279902037>
> Commit: `0068765c0d77086462af64c334d64c5e7ee8036b`
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=8279902037&attempt=2" target="_blank">Click here!</a>
> All cypress tests have passed 🎉🎉🎉

### CODE REVIEW

**Comment by KelvinOm** [comment]
> /ok-to-test tags="@tag.Widget"

Seems that in order to run tests with this tag, you need to change description and then remove + add `ok-to-test` label.

File: app/client/src/widgets/wds/WDSInputWidget/config/defaultsConfig.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { ResponsiveBehavior } from "layoutSystems/common/utils/constants";
+import { WDSBaseInputWidget } from "../../WDSBaseInputWidget";
+
+export const defaultsConfig = {
+  ...WDSBaseInputWidget.getDefaults(),
+  labelPosition: "top",
+  inputType: "TEXT",
+  widgetName: "Input",
+  version: 2,
```
Comment (riodeuno) [inline app/client/src/widgets/wds/WDSInputWidget/config/defaultsConfig.ts:9]: @KelvinOm @jsartisan - Is this value intentional?

**Pull request merged**